src/pages/EditMonitor.vue

```diff
@@ -377,20 +377,11 @@
                             <template v-if="monitor.type === 'sqlserver' || monitor.type === 'postgres' || monitor.type === 'mysql'">
                                 <div class="my-3">
                                     <label for="sqlConnectionString" class="form-label">{{ $t("Connection String") }}</label>
-
-                                    <template v-if="monitor.type === 'sqlserver'">
```

Comment:
Recently I did a PR to add support for DB2 (https://github.com/louislam/uptime-kuma/pull/3439) and I couldn't really understand the need for all these templates...
Glad to see this go 👍